inspirehep/dojson/common/base.py

```diff
@@ -329,9 +329,10 @@ def deleted_records2marc(self, key, value):
 @journals.over('urls', '^856.[10_28]')
 def urls(self, key, value):
     """URL to external resource."""
-    urls = []
+    urls = self.get('urls', [])
     value = utils.force_list(value)
     for val in value:
+        url_dict = {}
         if isinstance(val.get('y'), str):
             description = val.get('y')
         elif isinstance(val.get('y'), tuple):
```

Comment:
Please support also `list`. Then it would be `isinstance(val.get('y'), (list, tuple)):`